src/ansys/mapdl/core/mapdl_extended.py

```diff
@@ -375,14 +375,21 @@ def mpread(self, fname="", ext="", lib="", **kwargs):
         return self.input(fname)
 
     @wraps(_MapdlCore.cwd)
-    def cwd(self, *args, **kwargs):
+    def cwd(self, dirpath: str = "", **kwargs):
         """Wraps cwd."""
+        dirpath = str(dirpath)
+        if not (dirpath.startswith("'") and dirpath.endswith("'")) and "'" in dirpath:
+            raise MapdlRuntimeError(
+                'The CWD command does not accept paths that contain singular quotes "\'".'
+            )
+        kwargs["mute"] = False
+
         try:
-            output = super().cwd(*args, mute=False, **kwargs)
+            output = super().cwd(dirpath, **kwargs)
         except MapdlCommandIgnoredError as e:
             raise IncorrectWorkingDirectory(e.args[0])
 
-        self._path = args[0]  # caching
+        self._path = dirpath  # caching
```

Comment:
This should be extracted to another PR since it does not fit the content. And also add the proper tests.

Reply:
I agree, I will do so.

Reply:
Done with https://github.com/ansys/mapdl-cmd-conv/pull/164/commits/83fbfe0c680722f011d99ef75af4372de71e2424

Reply:
Done with https://github.com/ansys/pymapdl/pull/4060/commits/003337241ffac3aeceb249e6353b810aa44afd6e!

After having a closer look at it, I think the error was fixed after I made local modifications. If the tests passes, I don't think I will have to modify the ``cwd`` command.